generators/blank-marketplace/index.js

```diff
@@ -0,0 +1,89 @@
+const Generator = require("yeoman-generator");
+const chalk = require("chalk");
+const path = require("path");
+const mkdirp = require('mkdirp');
+
+
+
+module.exports = class extends Generator {
+  prompting() {
+    const prompts = [
+      {
+        type: "input",
+        name: "name",
+        message: "What is the name of your marketplace?"
+      },
+      {
+        type: "input",
+        name: "production",
+        message: "Set PRODUCTION url:"
+      },
+      {
+        type: "input",
+        name: "staging",
+        message: "Set STAGING url:"
+      },
+      {
+        type: "input",
+        name: "local",
+        message: "Set LOCAL url:"
+      }
+    ];
+
+    return this.prompt(prompts).then(props => {
+      this.props = props;
+    });
+  }
+
+  writing() {
+    const projectName = this.props.name.replace(/\s+/g, '-').toLowerCase();
```

Comment:
In second iteration we can use this to propose some defaults for urls :)

Reply:
Yes, I was thinking about it but then we would need to know the region